static/app/utils/analytics/workflowAnalyticsEvents.tsx

```diff
@@ -76,7 +76,8 @@ export type TeamInsightsEventParameters = {
   'issue_details.attachment_tab.screenshot_modal_download': {};
   'issue_details.attachment_tab.screenshot_modal_opened': {};
   'issue_details.attachment_tab.screenshot_title_clicked': {};
-  'issue_details.codecov_link_clicked': {};
```

Comment:
Removed analytic event from previous PR as I forgot to remove it (the event was moved, so it is not needed here)